.pre-commit-config.yaml

```diff
@@ -27,6 +27,8 @@ repos:
     rev: v1.9.4
     hooks:
       - id: biome-check
+        # args: [--unsafe]
```

Comment:
I'm not sure if this is a WIP or some debugging code.

Reply:
Thanks! I left that it because activating it temporarily may be useful when modernizing code, but I should either add a comment or remove it again.